### PR TITLE
Add `WAIT` examples to START/STOP sections and syntax tables to the Cypher adm commands

### PR DESCRIPTION
**Pull request by NataliaIvakina**
This is a big PR and I think it would be better to merge it after 5.20 release.

- Syntax tables were added to the modified pages
- Pages were restructured in line with the neo4j-admin commands section
- Usage examples were aligned

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1615.surge.sh

Updated pages:
- [Alter databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Create, start, and stop databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)

File: modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc [hunk 1]
```diff
@@ -16,10 +36,13 @@ DROP DATABASE customers
 ====
 Both standard databases and composite databases can be deleted using this command.
 ====
+======
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:39]:
```suggestion
```

File: modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc [hunk 2]
```diff
@@ -16,10 +36,13 @@ DROP DATABASE customers
...
 
 The `DROP DATABASE` command removes a database entirely.
 Therefore, it no longer shows up in the listing provided by the command `SHOW DATABASES`.
 
+.Example
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:44]:
```suggestion
```

File: modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc [hunk 3]
```diff
@@ -16,10 +36,13 @@ DROP DATABASE customers
...
+======
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:45]:
```suggestion
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1615.surge.sh

Updated pages:
- [Alter databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Create, start, and stop databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)

**Review by renetapopova — COMMENTED** [review]
That's a long one 😄. Thanks a lot for improving those pages. I added some comments related to the content, not your changes, but I think we'll need to look at it again.

**Comment by NataliaIvakina** [comment]
> That's a long one 😄. Thanks a lot for improving those pages. I added some comments related to the content, not your changes, but I think we'll need to look at it again.

yes, sure. Thanks a lot for reviewing

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1615.surge.sh

Updated pages:
- [Alter databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Create, start, and stop databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1615.surge.sh

Updated pages:
- [Alter databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Create, start, and stop databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1615.surge.sh

Updated pages:
- [Alter databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Create, start, and stop databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1615.surge.sh

Updated pages:
- [Alter databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Configuration parameters](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/configuration-parameters)
- [Create, start, and stop databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1615.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)

**Review by renetapopova — APPROVED** [review]

**Pull request merged**